### ISSUE TITLE
AutoComplete:#5298 fix expects queryMode="server" to be default, but is "null"

### ISSUE THREAD

**Issue by arnis87** [primefaces/primefaces#5596]
## 1) Environment
- PrimeFaces version: 6.2.27
- Does it work on the newest released PrimeFaces version? Version? yes

## 2) Expected behavior
`AutoComplete.queryMode` should default to "server" as per documentation

## 3) Actual behavior
`AutoComplete.queryMode` in latest elite release is `null `causing unexpected event broadcasts with `query=null`

## 6) Sample code
```java
public java.lang.String getQueryMode() {
	return (java.lang.String) getStateHelper().eval(PropertyKeys.queryMode, null);
}
```

**Comment by melloware** [comment]
@mertsincan This is an Elite release issue with 6.26 which is supposed to have this fix in it:
https://github.com/primefaces/primefaces/issues/5298

I am closing this issue as its an Elite issue and fixed correctly in master

Master has this
```java
    public String getQueryMode() {
        return (String) getStateHelper().eval(PropertyKeys.queryMode, "server");
    }
```

**Comment by arnis87** [comment]
So this will be fixed in elite .28?

**Comment by melloware** [comment]
That is up to @mertsincan but this tracker is only for Community issues not for Elite issues which is why I closed it.

**Comment by lagar84** [comment]
@mertsincan this will be fixed on 6.2.28?  I saw on logs that, on page load, autocomplete methods are called because of this bug on 6.2.27
